governance-and-automation/automated-workflows/policy-automation.md

```diff
@@ -0,0 +1,84 @@
+---
+description: This article explains how to set up and utilize the policy automation feature. 
+
+---
+
+# Policy Automation
+
+The Policy Automation feature **lets you create rules for auto-applying policies based on specific metadata and built-in properties**. 
+
+This makes managing your environment and keeping it secure easier and more efficient. With policy automation, **you can define rules in Syskit Point that will automatically apply policies** (for example, Access Review, Guest Users Expiration, Maximum Owners, etc.) to your newly created workspaces **without having to apply them manually**. It also means you'll need to create fewer provisioning templates to secure your environment. 
+
+{% hint style="important" %}
+**Please Note!**
+
+To activate rules for auto-applying policies, please make sure you have already set up your policies under the automated workflow section. [Follow this article for more information. ](../automated-workflows/set-up-automated-workflows.md)
+
+{% endhint %}
+
+
+## Create Rules
+
+To create rules for auto-applying policies, take the following steps:
+
+* Open your Syskit Point and **navigate to Settings** and under **Governance** select **Policy Automation (1)**
+* **Click the New Rule button (2)** and a pop-up will appear
+
+![Policy Automation - Create Rule](../../.gitbook/assets/policy-automation_policy-settings.png)
+
+To create the rule, fill out the following information, as requested on the pop-up:
+
+* Under the General section, fill out the following:
+  * **Rule Name (1)** - in this space, you can write your desired rule name
+  * **Description (2)** - this space can be used to describe the rule which you want to create
+* Under the **Priority section (3)**, set your desired level of priority for this rule by **selecting a number (4)** between 1-6 from the drop-down menu
+  * Priority will be **set as 1 by default** and is the number that signifies the highest priority, with 6 being the lowest priority
+  * This change also affects priorities on previously defined rules
+* When ready, **click Next to continue (5)**
+
+![Policy Automation - New Rule](../../.gitbook/assets/policy-automation_new-policy-first.png)
+
+
+* Under the **Add Conditions (1)** section, **set the conditions for your rule (2)** by using the drop-down menu
+  * Once you've set the condition, **click the Save Condition (3)** button
+  * Once saved, if you want to make changes **click the edit button** 
+  * Once saved, if you want to delete it, **click the delete button**
+* When you're ready, **click Next to continue (4)**
+
+![Policy Automation - New Rule - Add Condittions](../../.gitbook/assets/policy-automation_new-policy-second.png)
```

Comment:
na ovoj slici fali Sensitivity label jer na QA tenantu nije to podešeno (dodao sam si na todo listu)
napravio sam screenshot na svojem tenantu i stavio na QA site pa njega modificiraj da staviš ovdje :)

https://syskit.sharepoint.com/:i:/r/sites/QATeam/SysKit%20Docs/Images%20%26%20Snagit%20files/SysKit%20Point/governance-and-automation/automated-workflows/policy-automation/[policy-automation-conditions.png](https://syskit.sharepoint.com/:i:/r/sites/QATeam/SysKit%20Docs/Images%20%26%20Snagit%20files/SysKit%20Point/governance-and-automation/automated-workflows/policy-automation/policy-automation-conditions.png?csf=1&web=1&e=aWT4yS)?csf=1&web=1&e=aWT4yS